zbus/src/address/transport/mod.rs

```diff
@@ -75,6 +75,12 @@ impl fmt::Display for Transport<'_> {
     }
 }
 
+pub(crate) trait TransportTrait<'a> {
```

Comment:
Too bad `Tranport` name is already taken. I think the convention is `TransportImpl` though.

Reply:
ok